lib/util/http/jira.spec.ts

```diff
@@ -0,0 +1,33 @@
+import * as httpMock from '../../../test/http-mock';
+import * as hostRules from '../host-rules';
+import { JiraHttp, setBaseUrl } from './jira';
+
+describe('util/http/jira', () => {
+  const api = new JiraHttp();
+
+  it('throws error if setBaseUrl not called', async () => {
+    await expect(api.postJson('some-path')).rejects.toThrow(
+      TypeError('Invalid URL')
```

Comment:
shouldn't use that `new`?

```suggestion
      new TypeError('Invalid URL')
```

i prefer the constructor variant

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError/TypeError